src/view/com/composer/text-input/TextInput.web.tsx

```diff
@@ -297,15 +298,9 @@ export const TextInput = React.forwardRef(function TextInputImpl(
     style.lineHeight = style.lineHeight
       ? ((style.lineHeight + 'px') as unknown as number)
       : undefined
+    style.minHeight = webForceMinHeight ? 140 : undefined
     return style
-  }, [t, fonts])
-
-  React.useLayoutEffect(() => {
-    let node = editor?.view.dom
```

Comment:
editor.view.dom is actually referencing the same DOM node as the one that EditorContent renders. Which is why this change was made

Reply:
interesting, i'll need to have a look! i remember trying to do it there but it didn't work for some reason, maybe i made a mistake though.

Reply:
No worries, I did double-check on my end, but always good to have a second look!